contracts/utils/GenericSchemeMultiCallFactory.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity 0.5.17;
+
+import "../schemes/GenericSchemeMultiCall.sol";
+import "../schemes/SimpleSchemeConstraints.sol";
+import "@daostack/infra/contracts/votingMachines/GenesisProtocol.sol";
+
+/**
+ * @title 
+ */
+contract GenericSchemeMultiCallFactory {
+    uint8 public constant CUSTOM = 0;
+    uint8 public constant FAST = 1;
+    uint8 public constant NORMAL = 2;
+    uint8 public constant SLOW = 3;
+
+    function createGenericSchemeMultiCallSimple(
+        Avatar _avatar,
+        IntVoteInterface _votingMachine,
+        uint8 _voteParamsType,
+        uint256[11] memory _votingParams,
+        address _voteOnBehalf,
+        address[] memory _contractsWhiteList,
+        string memory _descriptionHash
+    ) public returns(address) {
```

Comment:
`public`  -> `external`

Reply:
I can't this gives a stack too deep error. But it doesn't really matter anyway...